pkg/lib/cockpit-components-modifications.jsx

```diff
@@ -71,34 +73,26 @@ class ModificationsExportDialog extends React.Component {
                     <Modal.Title>{ _("Automation Script") }</Modal.Title>
                 </Modal.Header>
                 <Modal.Body>
-                    <TabContainer id="basic-tabs-pf" defaultActiveKey="shell">
-                        <>
-                            <Nav bsClass="nav nav-tabs nav-tabs-pf" onSelect={this.handleSelect}>
-                                <NavItem eventKey="shell">
-                                    {_("Shell Script")}
-                                </NavItem>
-                                {this.props.ansible &&
-                                    <NavItem eventKey="ansible">
-                                        {_("Ansible Playbook")}
-                                    </NavItem>
-                                }
-                            </Nav>
-                            <TabContent animation>
-                                <TabPane eventKey="shell">
-                                    <pre>
-                                        {this.props.shell}
-                                    </pre>
-                                </TabPane>
-                                {this.props.ansible &&
-                                    <TabPane eventKey="ansible">
-                                        <pre>
-                                            {this.props.ansible}
-                                        </pre>
-                                    </TabPane>
-                                }
-                            </TabContent>
-                        </>
-                    </TabContainer>
+                    <Tabs activeKey={this.state.active_tab} onSelect={this.handleSelect}>
+                        <Tab eventKey="shell" title={_("Shell Script")}>
+                            <pre>
+                                {this.props.shell.trim()}
+                            </pre>
+                        </Tab>
+                        <Tab eventKey="ansible" title={_("Ansible Tasks")}>
```

Comment:
While this indeed goes into tasks/, the plural looks confusing here as it's really just one file. How about simply "Ansible" for the tab header, and explaining the details within the tab?